mungegithub/README.md

```diff
@@ -117,7 +117,7 @@ A valid approver is able to say `/approve` and mungegithub will add "approved" l
  
  ## FAQ
  
-* Why my pr hasn't been retest
+* Why my PR hasn't been retest
```

Comment:
This contradicts what is below. Do you mean : why my PR is not merged by the bot?
- missing one of the four labels - ...-
- has "do-not-merge" label
- tests failed.

Reply:
Retest means rerun pre-submit, if i remember correctly.

Reply:
Yeah, you guys are both right. First people are wondering why my PR isn't merged, while technically, in many case, it hasn't been re-test because of some following reasons. But, right, I am going to make it clearer.